### PR TITLE
Add Function Controller to installation process

### PR DESCRIPTION
**Pull request by magicmatatjahu**
**Description**

Changes proposed in this pull request:
- create temporary `resources/function-contoller-init` chart for installing needed by fun-con CRDs from cert-manager
- update `resources/function-contoller` chart
- optional installation of fun-con

**Related issue(s)**
https://github.com/kyma-project/kyma/issues/7142

### CODE REVIEW

File: resources/function-controller/templates/functions.crd.yaml [hunk 1]
```diff
@@ -27,10 +25,8 @@ spec:
   group: serverless.kyma-project.io
   names:
     kind: Function
-    listKind: FunctionList
     plural: functions
-    singular: function
-  scope: Namespaced
+  scope: ""
```
Comment (m00g3n) [inline resources/function-controller/templates/functions.crd.yaml]: I think we are missing the `Namespaced` value for scope.

From documentation of kubernetes 1.16 - [CustomResourceDefinitionSpec](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.16/#customresourcedefinitionspec-v1beta1-apiextensions-k8s-io)

>  Allowed values are `Cluster` and `Namespaced`. Default is `Namespaced`.

File: installation/resources/installer-cr-cluster-runtime.yaml.tpl [hunk 2]
```diff
@@ -24,14 +24,18 @@ spec:
       namespace: "kyma-installer"
     - name: "istio-kyma-patch"
       namespace: "istio-system"
-    #- name: "tekton-pipelines"
-    #  namespace: "tekton-pipelines"
+    - name: "tekton-pipelines"
```
[the remark this answers is not in the document]
Comment (magicmatatjahu) [reply installation/resources/installer-cr-cluster-runtime.yaml.tpl]: Yes, they will be disabled, but for testing I enabled they.

File: resources/function-controller-init/Chart.yaml [hunk 3]
```diff
@@ -0,0 +1,24 @@
+apiVersion: v1
+appVersion: 0.1.0
+description: Helm chart for installing needed CRDs for Function Controller
+name: function-controller-init
+version: 0.1.0
+keywords:
```
[the remark this answers is not in the document]
Comment (magicmatatjahu) [reply resources/function-controller-init/Chart.yaml]: Ok :)

File: resources/function-controller/templates/functions.crd.yaml (hunk 1 above)
Comment (magicmatatjahu) [reply resources/function-controller/templates/functions.crd.yaml]: I copy it from our folder with component source

**Pull request merged**